### ISSUE TITLE
Final project (creating Smooth zones 7x7 map)

### ISSUE THREAD

**Issue by Ethan-Hester** [leombastos/2023_AdvPA#62]
I am not able to run this chunk. I have tried restarting R, rewriting this chunk, and restarting my computer. It starts the process and just seems to freeze. I have tried everything and have let it run overnight to see if it would work through the issue. Any suggestions would be greatly appreciated. ![image](https://user-images.githubusercontent.com/122289190/234345763-777c96b8-9c70-40e1-ae0f-eaa27ea79a7b.jpeg)

**Comment by Ethan-Hester** [comment]
Update- literally 2 seconds after I posted this and after several hours of running this was produced: ![image](https://user-images.githubusercontent.com/122289190/234346355-3474fdc5-c10f-4c3f-a6bc-9e042be025c6.jpeg)

Still not sure why I am not able to produce a map...

**Comment by leombastos** [comment]
Ethan,

Can you share here the code you used to create `zone_s`?

**Comment by Ethan-Hester** [comment]
![image](https://user-images.githubusercontent.com/122289190/234606615-702e81f8-2f2f-468e-8d36-796f6f876b59.jpeg)

**Comment by leombastos** [comment]
@Ethan-Hester 
Between lines 137 and 138, you're missing the rounding step:
`mutate(cluster_f = round(cluster_f, 0))`

That should fix it, let me know how it goes.

**Comment by Ethan-Hester** [comment]
@leombastos Yes that worked! However now it will not let me combine mean pixel and cv pixel with pivot_longer. ![Uploading 234732087...]

**Comment by Ethan-Hester** [comment]
![image](https://user-images.githubusercontent.com/122289190/234732263-b8437912-fe2a-45b6-afd6-ad9619a791c9.jpeg)

**Comment by leombastos** [comment]
The error message is telling you something, check it closely on the line starting with "Error in tidy:: ..."

**Comment by Ethan-Hester** [comment]
But when I go back and watch the recording of your lecture. We did not specify anything more than what I have written. Am I missing a step?

**Comment by leombastos** [comment]
I'm not sure without seeing your code.

What is the error telling you? That the variables you are pivoting are not of the same class. One is numerical, the other is character.

They should be of the same class to be able to pivot.
Which one the two classes both of them should be?
Make a decision and change one of them so they both have the same class.

**Comment by thapashekhar** [comment]
@Ethan-Hester, How did you calculate mean_pixel and cv_pixel? Do we have enough data to calculate these variables?

**Comment by Ethan-Hester** [comment]
@thapashekhar No we do not have enough data to create cv or sd pixel. I was able to create mean pixel. But we would need 2-3 years worth of data to create the others.

**Comment by thapashekhar** [comment]
You are right, Thanks.
